### ISSUE TITLE
Default Configuration

### ISSUE THREAD

**Issue by nithin-mk** [JK-Flip-Flop96/powershell#3]
Hi, I have Windows Terminal and Powershell 7.4.2 installed. I've changed the Terminal's theme to Catpuccin Mocha as per the instructions in https://github.com/catppuccin/windows-terminal/blob/main/README.md. 

My question is how to use https://github.com/catppuccin/powershell to get a default, nice configuration for Catppuccin Mocha in Powershell for Windows Terminal? I did not quite understand the instructions given here in https://github.com/catppuccin/powershell/blob/main/README.md. 

In particular, I would like to fix the folder listing colours issue mentioned in https://github.com/catppuccin/windows-terminal/issues/11.
Thank you very much!

**Comment by nithin-mk** [comment]
I've completed the steps up to 4 in Usage section of README.md. But beyond that, I couldn't figure out how.

**Comment by sgoudham** [comment]
Hi 👋,

@nithin-mk I believe this repository allows the Catppuccin colours to be installed as a "library" to use within Powershell scripts. As a result, I'm not really sure what you mean by "default configuration" here.

With regards to the colours of the folders, I would need to try and configure it myself to understand how it's set up, but if it's only allowing ANSI colours/sequences then you wouldn't be able to use the hex codes from this library. That being said, it looks like the library does expose the colours in ANSI sequence format as shown on the [README](https://github.com/catppuccin/powershell/blob/202e1b363bff4b51542a903d9b5359dbd2f37340/README.md?plain=1#L59-L65).

@gravelfreeman Happy to accept a PR for revised instructions that simplify the wording and clarify the usage of the repository, but there's no need for that attitude here. Please align yourself with points 2. and 4. of our [Code of Conduct](https://github.com/catppuccin/.github/blob/main/CODE_OF_CONDUCT.md#community-guidelines).
